### PR TITLE
tel-53 - get tenant by public id 

### PR DESCRIPTION
**Pull request by AmitkumarEvry**
get tenant by public id
unit test cases

### CODE REVIEW

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 6fa46a443e82337c333d3091369f22252ec487e3
* Result: FAILED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=cc652eaa-ebf2-425a-b610-e286d57f4e6f%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/WWTelematics.TenantService.Tests/Application/TenantManagerTest.cs [hunk 1]
```diff
@@ -155,32 +153,50 @@ public void CanReturnSingleEntity()
...
+        public void CanReturnEmptyTenantList()
         {
-            var mockDbContext = GetPopulatedMockDbContext(MinimalValidData.Tenants);
+            List<Tenant> tenants = new List<Tenant>();
+            var mockDbContext = GetPopulatedMockDbContext(tenants);
 
-            TenantManager tenantManager = new TenantManager(logger, mockDbContext.Object);
-            ApplicationLoadBalancerResponse albResponse = tenantManager.GetTenantList();
+            TenantManager app = new TenantManager(logger, mockDbContext.Object);
+            ApplicationLoadBalancerResponse albResponse = app.GetTenantList();
```
Comment (NagarajNidagundi) [inline src/WWTelematics.TenantService.Tests/Application/TenantManagerTest.cs]: Change in all places "app" to "tenantManager"

File: src/WWTelematics.TenantService.Tests/Application/TenantManagerTest.cs [hunk 2]
```diff
@@ -155,32 +153,50 @@ public void CanReturnSingleEntity()
...
+        /// </summary>
+        [Fact]
+        public void CanNotReturnMissingTenant()
+        {
+            var mockDbContext = GetPopulatedMockDbContext(MinimalValidData.Tenants);
+
+            TenantManager app = new TenantManager(logger, mockDbContext.Object);
+            Guid testGuid = new Guid("00000000-0000-0000-0000-000000000000");
+
+            ApplicationLoadBalancerResponse albResponse = app.GetTenantByPublicId(testGuid);
```
Comment (NagarajNidagundi) [inline src/WWTelematics.TenantService.Tests/Application/TenantManagerTest.cs]: rename to "testPublicId".

File: src/WWTelematics.TenantService.Tests/Application/TenantManagerTest.cs (hunk 1 above)
Comment (AmitkumarEvry) [reply src/WWTelematics.TenantService.Tests/Application/TenantManagerTest.cs]: Changed as suggested

File: src/WWTelematics.TenantService.Tests/Application/TenantManagerTest.cs (hunk 2 above)
Comment (AmitkumarEvry) [reply src/WWTelematics.TenantService.Tests/Application/TenantManagerTest.cs]: Changed as suggested

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 0b8aef93de133c6087ba83b934c187fb206f3712
* Result: FAILED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=d8a6000b-aee7-44a6-8d19-ec581a1283b4%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 6452a00e0639b3b97881cc31367372d657485dcd
* Result: FAILED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=d5e39b94-1512-4472-890e-e7656797eb89%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 417b880cfc200a0c44a1df7ff8c1aa287a79b9aa
* Result: FAILED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=d4beee04-96a5-4d6d-96cd-361fc366dd45%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**